packages/tpetra/core/src/Tpetra_computeRowAndColumnOneNorms_def.hpp

```diff
@@ -127,6 +127,8 @@ computeLocalRowScaledColumnNorms_RowMatrix (EquilibrationInfo<typename Kokkos::A
   using mag_type = typename KAT::mag_type;
 
   auto rowNorms_h = Kokkos::create_mirror_view (result.rowNorms);
+
+  // DEEP_COPY REVIEW - NOT TESTED
```

Comment:
This is a working, but poor performing implementation which uses applies a function pointer to each row of the matrix via a getRow.

If we cared about the performance of the thing we'd just rewrite this all in the Kokkos-verse.

But as it is, those are device-to-host / host-to-device deep copy calls (in that order).